apps/backend/api/models/FundingRound.js

```diff
@@ -0,0 +1,506 @@
+        await FundingRound.distributeTokens(round, { transacting })
+        await round.save({ phase: FundingRound.PHASES.VOTING }, { transacting, patch: true })
+        Queue.classMethod('FundingRound', 'sendPhaseTransitionNotifications', { roundId: round.id, phase: FundingRound.PHASES.VOTING })
+        transitionCount++
+      }
+
+      // Transition from voting to completed
+      const votingClosingRounds = await FundingRound.query(q => {
+        q.where('deactivated_at', null)
+        q.whereNotNull('voting_closes_at')
+        q.where('phase', FundingRound.PHASES.VOTING)
+        q.where('voting_closes_at', '<=', now)
+      }).fetchAll({ transacting })
+
+      for (const round of votingClosingRounds.models) {
+        await round.save({ phase: FundingRound.PHASES.COMPLETED }, { transacting, patch: true })
+        Queue.classMethod('FundingRound', 'sendPhaseTransitionNotifications', { roundId: round.id, phase: FundingRound.PHASES.COMPLETED })
+        transitionCount++
+      }
+
+      return transitionCount
+    })
+  },
+
+  create: async function (attrs, userId) {
+    attrs.voting_method = attrs.voting_method || 'token_allocation_constant'
+
+    return await bookshelf.transaction(async transacting => {
+      const round = this.forge({ created_at: new Date(), updated_at: new Date(), ...attrs })
+      round.save({}, { transacting })
+
+      // Create the special chat room for this round
+      const topic = await Tag.findOrCreate('‡funding_round_' + round.id, { transacting })
+      await Tag.addToGroup({ group_id: attrs.group_id, tag_id: topic.id, isSubscribing: true, isChatRoom: true }, { transacting })
+
+      // Add creator as a participant
+      await FundingRound.join(round, userId, { transacting })
+
+      return round
+    })
+  },
+
+  find: function (id) {
+    if (!id) return Promise.resolve(null)
+    return FundingRound.where({ id }).query(q => {
+      q.where('deactivated_at', null)
+    }).fetch()
+  },
+
+  join: async function (roundOrId, userId, { transacting } = {}) {
+    if (!transacting) {
+      return bookshelf.transaction(async transacting => {
+        return await FundingRound.join(roundOrId, userId, { transacting })
+      })
+    }
+
+    const round = typeof roundOrId === 'object' ? roundOrId : await FundingRound.find(roundOrId, { transacting })
+    const roundId = round.id
+
+    if (!round) {
+      throw new GraphQLError('Funding Round not found')
+    }
+    if (!round.get('published_at')) {
+      throw new GraphQLError('Funding Round is not published')
+    }
+    let roundUser = await FundingRoundUser.where({ funding_round_id: roundId, user_id: userId }).fetch({ transacting })
+    if (!roundUser) {
+      roundUser = await FundingRoundUser.create({ funding_round_id: roundId, user_id: userId }, { transacting })
+      await round.save({ num_participants: round.get('num_participants') + 1 }, { transacting })
+
+      // XXX: don't send notifications for joining a funding round for now
```

Comment:
Still true?

Reply:
yes. i could remove but thought id leave in for now in case we want to add soon